spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -235,4 +235,13 @@
       expect(current_path).to eq profile_path
     end
   end
+
+  describe 'clicking the logo image during 2fa process' do
+    it 'signs the user out and returns them to the home page' do
+      user = build_stubbed(:user, :signed_up)
+      sign_in_user(user)
+      find("img[alt='login.gov']").click
+      expect(current_path).to eq root_path
```

Comment:
> Or, we might want to allow partially signed in users to access the root url without signing them out, in which case we also want to test that they are not signed out.

This part is a little bit confusing to me, could you explain it? Are you worried about the content at the root path changing (for example, we no longer have a sign in form there?). If that was the case, we would have to make (maybe significant) code changes to other parts of the codebase as well.

In the issue for this PR, you said:

>  For example, if you're signed in to github.com and you click the logo, it doesn't show you to the GitHub home page. It shows you a bunch of stuff related to your account because you're already signed in.

If the functionality we want is ‘returning the user to the root path doesn't sign them out’, that's is fine, but it is a different feature. I can write new code to reflect that if that’s the case. I'll ask product or UX for some clarity around this behavior.